src/crab/zone_domain.cpp

```
@@ -0,0 +1,1035 @@
+    } else {
+        rev_map_[vert] = lhs;
+    }
+    // Clear the old x vertex
+    havoc(lhs);
+    vert_map_.emplace(lhs, vert);
+
+    normalize();
+    CRAB_LOG("zones-split", std::cout << "---" << lhs << ":=" << e << "\n" << *this << "\n");
+}
+
+ZoneDomain ZoneDomain::narrow(const ZoneDomain& o) const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)